tests/test_geodist.py

```
@@ -1,27 +1,43 @@
+    geodesic_vincenty_inverse,
+    geodesic_vincenty,
+    great_circle,
+    great_circle_array,
+    geodist_dimwise,
+    geodist_dimwise_harvesine,
+)
 
 
-# Define test cases using pytest
+# ---------------------------------------------------------------------------
+# geodist – parametrized core tests
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)